src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -165,7 +160,8 @@ func runClientProxy(cfg *config.Config) {
 
 			proxiedsites.Configure(cfg.ProxiedSites)
 			configureStats(cfg, false)
-			client.Configure(cfg.Client)
+			hqfd = client.Configure(cfg.Client)
+			geolookup.Configure(hqfd.DirectHttpClient())
```

Comment:
I now continuously call geolookup.Configure so that if the hqfd changes, we get the latest.